docs/pipeline/eager/mpcdf.config

```diff
@@ -0,0 +1,11 @@
+# nf-core/configs: mpcdf eager specific configuration
```

Comment:
This isn't EVA anymore ;-)

Reply:
Ah yeah, this is a bonus because apparently I forgot to commit it in the previous profile PR :eyes: